app/src/main/java/org/sopt/and/ui/screen/myPage/viewmodel/MyScreenViewModel.kt

```diff
@@ -0,0 +1,35 @@
+package org.sopt.and.ui.screen.myPage.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+import org.sopt.and.data.UserManager
+import javax.inject.Inject
+
+@HiltViewModel
+class MyScreenViewModel @Inject constructor(
+    private val userManager: UserManager
+) : ViewModel() {
+    private val _profileName = MutableStateFlow("프로필1")
+    val profileName: StateFlow<String> = _profileName
+
+    init {
+        loadUserProfile()
+    }
+
+    private fun loadUserProfile() {
+        viewModelScope.launch {
+            _profileName.value = userManager.getUserEmail() ?: "프로필1"
+        }
+    }
```

Comment:
아래와 같이 바꿀 수 있을 것 같아요!

```suggestion
    private val _profileName = MutableStateFlow(userManager.getUserEmail() ?: "프로필1")
    val profileName: StateFlow<String> = _profileName
```

Reply:
와 코드가 훨씬 간결해지네요.. 동기 비동기 상황에 따라서 필요한 코드에 대해서 공부해보겠습니다. 감사합니다!